en/docs/references/ballerina-expression-syntax.md

```diff
@@ -36,9 +36,9 @@ A signed integer is an integer with the  `+` or `-` character in the front, indi
   +2020
   ```
 
-### Decimal Floating-Point Number
+### Decimal floating-point number
 
-A decimal floating-point number is a number with a decimal point. You have the option to use `+` or `-` characters in the front to indicate the sign.
+A decimal floating-point number is a number with a decimal point. You can precede the numbers with the + or - characters to indicate the sign.
```

Comment:
`+ or -` is not highlighted in this case. But it is in the `Signed integer` topic. We should follow the same convention everywhere.  As per the comment https://github.com/wso2/docs-choreo-dev/pull/90/files#r636775634 I feel the backticks are the best option here.

Reply:
Since this is about decimal numbers, if we format + and - in bold it will get unnecessarily highlighted more than the decimal aspect. Whereas in Signed integer, we have to highlight + and - . 
Agree that backticks are the best option here.